openerp/addons/base/security/ir.model.access.csv

```diff
@@ -28,7 +28,7 @@
 "access_ir_sequence_group_system","ir_sequence group_system","model_ir_sequence","group_system",1,1,1,1
 "access_ir_sequence_date_range_group_user","ir_sequence_date_range group_user","model_ir_sequence_date_range","group_user",1,0,0,0
 "access_ir_sequence_date_range_group_system","ir_sequence_date_range group_system","model_ir_sequence_date_range","group_system",1,1,1,1
-"access_ir_translation_all","ir_translation all","model_ir_translation",,1,1,1,1
+"access_ir_translation_all","ir_translation all","model_ir_translation",,1,0,0,0
```

Comment:
Issue: a user without write access simply cannot edit translations, since the "Edit" button is hidden...